src/aws-lambda/bot-intent-recommendations/bundle.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
general comment on the bundle.sh and stage.sh.
It's the same for every Lambda function, maybe some opportunities to factor some code?

Reply:
Yes, there is a growing need to simplify how the growing number of lambdas are deployed. SAM is probably the answer.